spec/views_test/a_user_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  before(:each) do
+    DatabaseCleaner.clean
+    @user = User.create(
+      name: 'zubair',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'full-stack developer.',
+      posts_counter: 0
+    )
+    visit root_path
+  end
+
+  it 'should redirect to user show page when a name is clicked' do
+    user = User.find(@user.id)
+    click_link(user.name)
+    expect(page.current_path).to eql(user_path(id: user.id))
+  end
```

Comment:
- [ ] All the tests are passing except this one. Please fix this because it throws an error 
![image](https://user-images.githubusercontent.com/45460192/199500768-0dc32cd2-6b5a-41b9-86b7-181ee34e42d0.png)

When I comment this test, everything else passes.
![image](https://user-images.githubusercontent.com/45460192/199501269-48371d37-2b4d-4db0-9caf-eeb85601dfcc.png)

Reply:
fixed